ios/FluentUI/Tab Bar/TabBarView.swift

```diff
@@ -191,13 +191,43 @@ open class TabBarView: UIView, TokenizedControlInternal {
         }
     }
 
+    private class CustomTabBarItemTokens: TabBarItemTokens {
+        var tabBarTokens: TabBarTokens = .init()
```

Comment:
I don't think we should be initializing this to a default value here. Presumably that will cause the `required init()` below to fail, but we can work around that by having it throw an error (since no one should be calling it anyway):

```swift
@available(*, unavailable)
required init() {
    preconditionFailure("init() has not been implemented")
}
```